src/lib/Component/Content/ContentEditMetaFieldsComponent.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Component\Content;
+
+use Ibexa\Contracts\AdminUi\Component\Renderable;
+use Ibexa\Contracts\Core\SiteAccess\ConfigResolverInterface;
+use Twig\Environment;
+
+class ContentEditMetaFieldsComponent implements Renderable
+{
+    private const NO_CONTENT = '';
+
+    private Environment $twig;
+
+    private ConfigResolverInterface $configResolver;
+
+    public function __construct(
+        Environment $twig,
+        ConfigResolverInterface $configResolver
+    ) {
+        $this->twig = $twig;
+        $this->configResolver = $configResolver;
+    }
+
+    /**
+     * @param array<string, mixed> $parameters
+     *
+     * @return string
+     */
+    public function render(array $parameters = []): string
+    {
+        /** @var \Ibexa\Contracts\Core\Repository\Values\ContentType\ContentType $contentType */
+        $contentType = $parameters['content_type'];
+
+        $metaFieldTypeIdentifiers = $this->getMetaFieldTypeIdentifiers();
+
+        if (empty($metaFieldTypeIdentifiers)) {
+            return self::NO_CONTENT;
+        }
+
+        $parameters['meta_fields'] = [];
+        foreach ($metaFieldTypeIdentifiers as $identifier) {
+            $fields = $contentType->getFieldDefinitionsOfType($identifier);
+            $parameters['meta_fields'] += array_column($fields->toArray(), 'identifier');
```

Comment:
Is this += really intended here? This would mean that keys existing in left hand side would be kept. If that was the intention then ok, just making sure ;)

Reply:
I think it's safe here because Field Definition identifiers are unique in Content Type.